ee/vellum_ee/workflows/display/nodes/vellum/api_node.py

```diff
@@ -31,6 +31,19 @@ class BaseAPINodeDisplay(BaseNodeDisplay[_APINodeType], Generic[_APINodeType]):
         APINode.authorization_type,
     }
 
+    # TODO: Only serialize attributes not inputs
```

Comment:
[oos] Yea for context, our long term goal is to eventually get all of these out of this array and as attributes. We've made good progress on this in the prompt nodes case